style/haskell/README.md

```diff
@@ -5,13 +5,16 @@ Haskell
 
 * Break long expressions before operators or keywords.
 * Break long type signatures between arguments.
+* Don't use a blank line between pragma and module statements.
```

Comment:
There's absolutely no reason whatsoever, just something to decide for
consistency.

To me, having the space just looks weird, like putting a space between a
method and the comment documenting it.

Just based on the code I see around, no space is more common. No idea what
the real numbers could be though.
On Feb 13, 2015 8:36 PM, "Blake Williams" <notifications@github.com> wrote:

> In style/haskell/README.md
> <https://github.com/thoughtbot/guides/pull/305#discussion_r24710363>:
>
> > @@ -5,13 +5,16 @@ Haskell
> >
> >  * Break long expressions before operators or keywords.
> >  * Break long type signatures between arguments.
> > +* Don't use a blank line between pragma and module statements.
>
> I don't have much of an opinion on it, but why no blank line? I usually
> add one when I write Haskell.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/guides/pull/305/files#r24710363>.
>